src/main/scala/com/amazon/deequ/analyzers/NonSampleCompactor.scala

```diff
@@ -0,0 +1,47 @@
+package com.amazon.deequ.analyzers
+
+import scala.collection.mutable.ArrayBuffer
+import scala.reflect.ClassTag
+import scala.util.Random
+
+/**
+ * A quantile sketcher whose output is half the size of its input
+ *
+ *
+ * @tparam T type of the items being sketched. There should an ordering
+ *           over this item type
+ */
+class NonSampleCompactor[T]()
+                           (implicit ordering: Ordering[T],
+                            ct: ClassTag[T]) extends Serializable {
+
+  var numOfCompress = 0
+  var offset = 0
+  var buffer: ArrayBuffer[T] = ArrayBuffer[T]()
+  private def findOdd(items:Int): Option[T] = (items%2) match {
```

Comment:
please add an empty line before the function definition and spaces around %

Reply:
Sure.